robot_rodeo_gym/EntityPose pose
---
bool success